examples/codestral-code-interpreter-ts/README.md

```diff
@@ -0,0 +1,44 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools for code execution yet, so in this Python example, we added the code interpreting capabilities. We are going to build an AI agent that performs data analysis tasks on provided data, in a form of a csv file.
+
+
+# Installation
+
+First, cone the E2B Cookbook repository, and navigate to the correct directory.
+
+
+## Install dependencies
+
+Ensure all dependencies are installed via `npm install`.
+
+- `@mistralai/mistralai`
+- `@e2b/code-interpreter`
+- `dotenv`
```

Comment:
Remove this:
1. it's not all dependencies used anyway
2. No reason to enumerate deps